utils/src/commonMain/kotlin/com/badoo/reaktive/utils/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.utils
+
+const val NANOS_IN_MICROS = 1_000L
+const val NANOS_IN_MILLIS = 1_000_000L
```

Comment:
I believe the last word should be singular: `NANOS_IN_MICRO`, `NANOS_IN_MILLI`, `MICROS_IN_MILLI`. I have googled both variants and this one is being used more often.